assets/js/lib/filter/index.js

```diff
@@ -0,0 +1,4 @@
+const regularizeString = (str) => str.normalize().trim().toLowerCase();
+
+export const foundStringNaive = (str = '', substring = '') =>
```

Comment:
I don't find `foundStringNaive()` a very indicative name, maybe something like `containsSubstring()`?

Reply:
You're right. That name sounds so much better!